submissions/JuliaVorotchenko/Popup-task/img/account.svg

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
You have so many svg icons and all of them are separated files. Did you consider to use CSS Sprites?

http://i.imgur.com/3vDKqSz.png

Reply:
I have used sprites,  thank you very much for idea, even didn't know about that before))